webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/jdbc/EventLogJdbcRepository.scala

```diff
@@ -195,8 +195,13 @@ class EventLogJdbcRepository(
     transactIOResult(s"Error when retrieving event logs for change request '${xpath}'")(xa => (for {
       entries <- HC.stream[(String, String, EventLogDetails)](q, param, 512).compile.toVector
     } yield {
-      entries.map { case (crid, tpe, details) =>
-        (ChangeRequestId(crid.substring(1, crid.length()-1).toInt), toEventLog((tpe, details)) )
+      entries.flatMap { case (crid, tpe, details) =>
+        val optId = try {
+          Some(crid.filter(_ == '"').toInt)
+        } catch {
```

Comment:
I got a number format exception for a case where there was two double quotes - not sure why it happens.

Reply:
I'm not sure to understand what happen here `crid` is a `Collection` or a `String` ?

Reply:
A string is a collection of Char :smile: 

But here you are keeping " in crId, it should be filterNot I think

Reply:
yes, how was it able to find result ? Perhaps I missed a compilation step or whatever